README.md

```diff
@@ -29,15 +28,17 @@ pip install extremeflash
 
        ```commandline
        extremeflash --local-ip 192.168.1.70/24 \
-       -i ~/Downloads/openwrt-22.03.3-mpc85xx-p1020-extreme-networks_ws-ap3825i-initramfs-kernel.bin \
+       --model AP3710 \
+       -i ~/Downloads/openwrt-23.05.0-mpc85xx-p1020-enterasys_ws-ap3710i-initramfs-kernel.bin \
        -j ~/Downloads/openwrt-22.03.3-mpc85xx-p1020-enterasys_ws-ap3710i-squashfs-sysupgrade.bin
```

Comment:
Firmware should have matching versions I guess.
```suggestion
       -j ~/Downloads/openwrt-23.05.0-mpc85xx-p1020-enterasys_ws-ap3710i-squashfs-sysupgrade.bin
```
E.g. both openwrt-23.05.0

Reply:
Not necessarily. It should illustrate that you can load a random initramfs with a random sysupgrade. 
This was needed to boot some devices in the past. 

Do you think it is confusing as-is?

Reply:
I think you should give a hint that the versions don't have to match.
This is not mentioned in the openwrt wiki, so I expected that they must match (since the linked firmware has the same version).

Reply:
added